LoopCaregiver/LoopCaregiver/Views/Actions/CarbInputView.swift

```diff
@@ -104,10 +104,28 @@ struct CarbInputView: View {
             }
             
```

Comment:
I see why you used `.onTapGesture` rather than the Button's action block. SwiftUI forms seem to not respect the button bounds based on [this](https://stackoverflow.com/questions/57947581/why-buttons-dont-work-when-embedded-in-a-swiftui-form) which you may have seen. Your solution works well though.